connectors/class-connector-buddypress.php

```diff
@@ -1095,12 +1133,14 @@ public function field_group_action( $group, $action, $meta = array(), $message =
 	 * @param object $group  Field group.
 	 */
 	public function callback_xprofile_group_after_save( $group ) {
-		global $wpdb;
-		/**
-		 * A bit hacky, due to inconsistency with BP action scheme,
-		 * see callback_xprofile_field_after_save for correct behavior.
-		 */
-		$action = ( $group->id === $wpdb->insert_id ) ? 'created' : 'updated';
+		// Bail if the field group's action wasn't captured.
+		if ( is_null( $this->incoming_field_group_action ) ) {
+			return;
+		}
+
+		$action = $this->incoming_field_group_action;
+		unset( $this->incomming_field_group_action );
```

Comment:
@ivankruchkoff I below this is a better way to capture the incoming field action